gems/sorbet-runtime/test/rubocop-shims.rbi

```diff
@@ -0,0 +1,5 @@
+# typed: __STDLIB_INTERNAL
+
+module RuboCop
+  class RakeTask; end
+end
```

Comment:
Do we really need this? Do we typecheck the `Rakefile`? Or is `RakeTask` used somewhere else?

Reply:
I think since the rakefile isn't `typed: ignore` we at least need to be able to resolve constants?

Reply:
The Rakefile doesn't have a `.rb` extension so it is already ignored.

Reply:
Huh... I don't know then, but I did get a failure before adding this.